.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "laser"]
+	path = laser
+	url = git@github.com:PaperFanz/laser.git
```

Comment:
Not sure I would include the submodule of the assembler as a dependency, you can provide the .obj files already compiled and if the user wants to install an assembler let them do that themselves. This can add complexity to the repo.

Reply:
Doesn't it make the repo "easier to use"? 

The user simply has to follow the readme instructions (coming soon!) and the use the provided makefile; which will compile all the needed dependencies and everything "just works"?

Plus, the sub module can always be ignored, right?

Reply:
its ok it's a decision, that's just my opinion but you can check later if it's useful or not.